docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -324,17 +334,17 @@ The API returns the following response:
       "feature_states": [],
       "include_global_state": true,
       "state": "SUCCESS",
-      "start_time": "2020-07-06T21:55:18.129Z",
-      "start_time_in_millis": 1593093628850,
-      "end_time": "2020-07-06T21:55:18.129Z",
-      "end_time_in_millis": 1593094752018,
-      "duration_in_millis": 0,
+      "start_time": "2020-07-06T21:55:18.130Z",
+      "start_time_in_millis": 1593093628851,
```

Comment:
Fixed up the example values to a. actually make sense and be consistent across examples (not necessary but I found it nicer for readers) but also so that i have different timestamps/durations on all examples so that the test body replacements work properly.